docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-19_remove-sample-data-dir.txt

```diff
@@ -0,0 +1 @@
+* The deprecated configuration variable ```SAMPLE_DATA_DIR``` has been removed in favour of user installation of the ```iris_sample_data``` package.
```

Comment:
@corinnebosley where has `SAMPLE_DATA_DIR` been removed from? That is, could you add a reference to `iris.config` in here please?